src/index.tsx

```diff
@@ -1,14 +1,13 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import Main from "Main";
-import { UserProvider } from "context/UserProvider";
 import "assets/css/global.css";
 
-const root = ReactDOM.createRoot(document.getElementById("root"));
+const root = ReactDOM.createRoot(
+  document.getElementById("root") as HTMLElement
+);
```

Comment:
이렇게 표현해주는게 조금 더 정확하겠네요~
as assertion없이 if문으로 표현해줘도 괜찮을 것 같아요!
```suggestion
const root = ReactDOM.createRoot(
  document.getElementById("root") as HTMLDivElement
);
```